### PR TITLE
[RFR] Only use REST API for GET/POST of Advanced Settings

### PR DESCRIPTION
**Pull request by jawatts**
REST API is available in 5.9, use this for the GET/POST of advanced settings instead of rails.

{{ pytest:  cfme/tests/platform/test_advanced_config.py --use-provider complete }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ebe18c750c9a94024585303dc47ef4cd439e392e:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 2089:5: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/containers/test_blacklisted_container_events.py [hunk 1]
```diff
@@ -69,6 +68,18 @@
 }
 
 
+def restore_advanced_settings(appliance):
+    """Restores the Advanced Settings config"""
+    if appliance.version < '5.10':
```
Comment (john-dupuy) [inline cfme/tests/containers/test_blacklisted_container_events.py]: Why not use `VersionPicker` here?

File: cfme/tests/containers/test_blacklisted_container_events.py [hunk 2]
```diff
@@ -69,6 +68,18 @@
 }
 
 
+def restore_advanced_settings(appliance):
```
Comment (john-dupuy) [inline cfme/tests/containers/test_blacklisted_container_events.py]: Will `update_advanced_settings` be modified in the future to behave in this way? Or is this a temporary workaround?

File: cfme/tests/containers/test_blacklisted_container_events.py (hunk 1 above)
Comment (jawatts) [reply cfme/tests/containers/test_blacklisted_container_events.py]: I could but VersionPicker is generally used when you do not have the appliance object.

File: cfme/tests/containers/test_blacklisted_container_events.py (hunk 2 above)
Comment (jawatts) [reply cfme/tests/containers/test_blacklisted_container_events.py]: In what way are you referring to here?

Comment (john-dupuy) [reply cfme/tests/containers/test_blacklisted_container_events.py]: Nevermind, I see that here you are only changing a single setting that depends on the version of the appliance.

**Pull request merged**